demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -283,6 +229,11 @@ export class DemoMeetingApp
   static readonly DATA_MESSAGE_TOPIC: string = 'chat';
   static readonly DATA_MESSAGE_LIFETIME_MS: number = 300_000;
 
+  // Currently this is the same amount as max send videos so we label the check box 'enable'
+  // rather then 'reduce', but technically pagination is always enabled.
```

Comment:
This comment is a little bit confusing to me. Can you help me understand it better?